pootle/apps/reports/views.py

```diff
@@ -501,38 +503,43 @@ def get_daily_activity(user, scores, start, end):
     current_day_score = 0
     translated_group = {}
     reviewed_group = {}
+    suggested_group = {}
     for score in scores:
         score_time = make_naive(score.creation_time)
         date = score_time.date()
 
         translated, reviewed = score.get_paid_wordcounts()
-        if translated is not None or reviewed is not None:
+        suggested = score.get_suggested_wordcount()
+
+        if any(map(lambda x: x is not None, [translated, reviewed, suggested])):
```

Comment:
I think repeating 3 times `x is not None` is preferable to use some functions that make you think a bit more. If you have any plan to add more and more items to check, then I agree that any-lambda is the best approach to keep line length under control, but I don't think that is the case.

Personally I try to avoid `lambda` as much as possible.